docker-compose.test.yml

```diff
@@ -1,12 +1,12 @@
-version: '2.1'
+version: '3'
 services:
+  redis:
+    image: redis:4.0.9-alpine
```

Comment:
Redis in production is still on 3.x, this would be a mismatch with what's i use there. Do they have a 3.2-alpine?

Reply:
Should note, 4.x is not available to us yet on Elasticache.

Reply:
yep, there's a 3.2, I'll update